src/style.css

```diff
@@ -0,0 +1,119 @@
+@import url(./fonts/fonts.css);
+@import url(./reset/reset.css);
+@import url(./var/var.css);
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
Add font-family to html and don't repeat this prop further. Why you need flex in this case? Block elements by default located one below the other.